### ISSUE TITLE
Missing support for Node LTS 10.13.0

### ISSUE THREAD

**Issue by maximffs** [Azure/azure-functions-nodejs-worker#144]
#### Investigative information
On October 30th 2018, Node updated their LTS to v10.13.0 and current release to 11.x. 
Please see more details here: 
https://github.com/nodejs/Release#release-schedule

I ran it on a sample new Function App and HttpTrigger1 that is generated from the portal (no custom code at all)

Trying to set this function app to run using the LTS version 10.13.0 (by updating the app setting WEBSITE_NODE_DEFAULT_VERSION to 10.13.0) breaks the runtime and I'm getting this error:

```
Error:

Function (HttpTrigger1) Error: The system cannot find the file specified.
Session Id: 980dfd90b9c144adbfc2f4d3234f8a25

Timestamp: 2018-11-01T23:48:15.878Z
```

The only two values that work for this setting are "8.11.1" and "10.6.0".

Looking briefly in your code, I saw that those are the only two values you define in the appveyor.yml-> environment->matrix definition.

Also in nodejsWorker.js you set:
var ACTIVE_LTS_VERSION = "v8";
var CURRENT_BRANCH_VERSION = "v10";

which now should probably be updated to handle v10 and v11.

**Comment by ColbyTresness** [comment]
We'll definitely look to update to 10.13, but note that we're not locked into the same release cycle as Node, so we never planned to take this on October 30th. (As an aside, we never plan to support Node 11.)

Adding @mhoeger to fix the references to CURRENT in our code, that should definitely be switched as 10 is LTS.

**Comment by ColbyTresness** [comment]
Leaving this open to track the code fixes, opened #145 to track the actual version update.
